src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -138,11 +138,23 @@ FixedwingPositionControl::parameters_update()
 	param_get(_parameter_handles.pitch_limit_min, &(_parameters.pitch_limit_min));
 	param_get(_parameter_handles.pitch_limit_max, &(_parameters.pitch_limit_max));
 	param_get(_parameter_handles.roll_limit, &(_parameters.roll_limit));
-	param_get(_parameter_handles.throttle_min, &(_parameters.throttle_min));
-	param_get(_parameter_handles.throttle_max, &(_parameters.throttle_max));
-	param_get(_parameter_handles.throttle_idle, &(_parameters.throttle_idle));
-	param_get(_parameter_handles.throttle_cruise, &(_parameters.throttle_cruise));
+
+	float v = 0.0f;
+
+	param_get(_parameter_handles.throttle_idle, &v);
+	_parameters.throttle_idle = constrain(v, 0.0f, 1.0f);
+
+	param_get(_parameter_handles.throttle_min, &v);
+	_parameters.throttle_min = constrain(v, 0.0f, 1.0f);
```

Comment:
should we consider 3d planes here? (negative thrust)

Reply:
There are a number of changes required to support that.